### PR TITLE
Make FC041 smarter about CURL usage

### PR DESCRIPTION
**Pull request by odcinek**
Closes https://github.com/acrmp/foodcritic/pull/305, closes https://github.com/acrmp/foodcritic/issues/240

### CODE REVIEW

**Comment by odcinek** [comment]
@lamont-granquist @jaymzh I'll need your eyes on the test cases

File: features/041_check_raw_download.feature [hunk 1]
```diff
@@ -19,8 +19,13 @@ Feature: Check for direct usage of curl or wget
     When I check the cookbook
     Then the execute resource used to run curl or wget commands warning 041 <display> be displayed
   Examples:
-    | command                                 | display     |
-    | which foo                               | should not  |
-    | curl 'http://example.org/'              | should      |
-    | wget 'http://example.org/'              | should      |
-    | mkdir foo && wget 'http://example.org/' | should      |
+    | command                                    | display     |
```
Comment (jaymzh) [inline features/041_check_raw_download.feature]: command?

File: features/041_check_raw_download.feature [hunk 2]
```diff
@@ -19,8 +19,13 @@ Feature: Check for direct usage of curl or wget
...
+    | which foo                                  | should not  |
```
Comment (jaymzh) [inline features/041_check_raw_download.feature]: this is an odd example. should this be 'which curl' or 'which get' ?

File: features/041_check_raw_download.feature [hunk 3]
```diff
@@ -10,17 +10,25 @@ Feature: Check for direct usage of curl or wget
...
     Then the execute resource used to run curl or wget commands warning 041 <display> be displayed
   Examples:
-    | command                                 | display     |
-    | which foo                               | should not  |
-    | curl 'http://example.org/'              | should      |
-    | wget 'http://example.org/'              | should      |
-    | mkdir foo && wget 'http://example.org/' | should      |
+    | command                                    | display     |
+    | which curl                                 | should not  |
+    | sudo wget                                  | should not  |
```
Comment (jaymzh) [inline features/041_check_raw_download.feature]: sudo wget http://example.org/ *should*

Comment (odcinek) [inline features/041_check_raw_download.feature]: good catch sir

**Pull request merged**